### PR TITLE
Feature 376

### LINKED ISSUE #376: As a guardian i would like "instillinger" to shown the chosen setting so that i know what setting has been chosen.
**Describe the solution you'd like**
Under "instillinger"(the little cog, top right) at the weekplanner overview, it would be nice to actually seen what settings that has been chosen. 
In addition it would be nice when you chose a setting (eks. "vis kun i dag") it automatically jump back to  the menu. 

**Describe alternatives you've considered**
None

**Additional context**

### PR DESCRIPTION
**Pull request by jbjarn17**
Fixes #376

### CODE REVIEW

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw608f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [x] All new functionality is covered by tests
- [x] It is understandable, by reading the tests, what they do
- [x] The tests covers sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [x] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [x] There is well written code in this. Remember to tell the developer!

**Review by mads-bo — COMMENTED** [review]
@aau-giraf/sw610f20 you have been assigned this. Here's your checklist.
Please use the following checklist. Feel free to check it off as you go along.

**Code Design**

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems
- [ ] The code does it introduce functionality that is not necessary for solving the problem

**Code Readability and Maintainability**

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no "hack", or obscure solution
- [x] All new functionality is covered by tests
- [x] It is understandable, by reading the tests, what they do
- [x] The tests covers sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing
- [ ] Error messages in the code are understandable
- [ ] Confusing sections of code are covered by comments

**Functionality**

For this part, open the app, and test:

- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Points for Discussion and Reflection**

- [ ] UI messages, that the user might meet, are actually helpful
- [x] There are no errors that are bad practice. E.g. hard-coding a test database connection
- [x] There is well written code in this. Remember to tell the developer!

**Comment by MrPekar98** [comment]
The text in the settings screen right of 'Antal dage' is one action behind. When I click on 'Vis kun i dag', the text to the right of 'Antal dage' shows what I selected previously. If I choose 'Vis mandag til fredag', the text to the right of 'Antal dage' shows 'Vis kun i dag'.

**Review by MrPekar98 — APPROVED** [review]

File: test/screens/settings_screens/number_of_days_selection_screen.dart [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  testWidgets('Has option, Vis mandag til søndag', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(home: NumberOfDaysScreen(user)));
+    await tester.pumpAndSettle();
+    expect(find.text('Vis mandag til søndag'), findsOneWidget);
+  });
+
+  testWidgets('Has only one selected option', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(home: NumberOfDaysScreen(user)));
+    await tester.pumpAndSettle();
+    expect(find.byIcon(Icons.check), findsOneWidget);
```
Comment (kristiantg) [inline test/screens/settings_screens/number_of_days_selection_screen.dart:95]: Skal den her test (og de forrige) kaldes done(); på? Eller er det ligemeget? Jeg har set at nogle af de andre tests altid har done();.

Comment (MrPekar98) [reply test/screens/settings_screens/number_of_days_selection_screen.dart:95]: Det plejer kun at være i bloc tests, hvor der anvendes async tests.

**Review by kristiantg — APPROVED** [review]

**Pull request merged**